packages/dnspyex.vm/tools/chocolateyinstall.ps1

```diff
@@ -7,8 +7,9 @@ try {
 
   $zipUrl = "https://github.com/dnSpyEx/dnSpy/releases/download/v6.5.0/dnSpy-netframework.zip"
   $zipSha256 = "5962e3cca902e650c61050e2a879af58b78eec91288b7a7b77a7bc761424a0ed"
-  $toolDir = (VM-Install-From-Zip $toolName $category $zipUrl -zipSha256 $zipSha256)[0]
+  VM-Install-From-Zip $toolName $category $zipUrl -zipSha256 $zipSha256
```

Comment:
This function should return the `$toolDir` as documented in this comment in `$common`:
> # This functions returns $toolDir (outputed by Install-ChocolateyZipPackage) and $executablePath

If the previous PR got rid of that, we should bring it back instead of having to recalculate `$toolDir`. Other packages use this as well.